### PR TITLE
Add a link to estargz.kontain.me

### PR DESCRIPTION
**Pull request by ktock**
This commit adds a link to `estargz.kontain.me`.

Thanks @imjasonh !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,16 +111,42 @@ Hello World!
...
+This section describes some of them.
 
-In this section, we introduce `ctr-remote` command for converting images into eStargz with optimization for reading files.
+### Trying pre-built eStargz images
+
+You can try our pre-converted eStargz images on ghcr.io listed in [Trying pre-converted images](/docs/pre-converted-images.md).
+
+### Registry-side conversion with `estargz.kontain.me`
+
+You can convert arbitrary images into eStargz on the registry-side, using [`estargz.kontain.me`](https://github.com/imjasonh/kontain.me/tree/012b357a5e8f4a301e929c5739a4905b0e7bad86/cmd/estargz).
```
Comment (imjasonh) [inline README.md:129]: You can link to https://estargz.kontain.me and it'll redirect to the latest README.md, or theoretically in the future some other better landing page.

File: README.md [hunk 2]
```diff
@@ -111,16 +111,42 @@ Hello World!
...
-You can also try our pre-converted images listed in [Trying pre-converted images](/docs/pre-converted-images.md).
+There are several ways to achieve that.
+This section describes some of them.
 
-In this section, we introduce `ctr-remote` command for converting images into eStargz with optimization for reading files.
+### Trying pre-built eStargz images
+
+You can try our pre-converted eStargz images on ghcr.io listed in [Trying pre-converted images](/docs/pre-converted-images.md).
+
+### Registry-side conversion with `estargz.kontain.me`
```
Comment (imjasonh) [inline README.md:127]: Can you include some warning about stability/security of relying on my weekend side project? It's probably best used to easily try it out on whatever image you want, but if you start to rely on it in production you should either copy images to your own real registry, or build them yourself with `ctr-remote` as described below.

**Review by imjasonh — APPROVED** [review]
lgtm! Just a couple small comments.

File: README.md (hunk 1 above)
Comment (ktock) [reply README.md]: Thank you for sharing the link! fixed docs to use that.

File: README.md (hunk 2 above)
Comment (ktock) [reply README.md:127]: Thank you for the suggestion. added an warning message to README. LGTY?

Comment (imjasonh) [reply README.md:127]: Looks great! 👍

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**